pkg/url/url.go

```diff
@@ -20,13 +20,13 @@ import (
 )
 
 // Get returns URL defination for given URL name
```

Comment:
can you please change this to definition

Reply:
ok